AWSS3/AWSS3TransferUtility.m

```diff
@@ -1071,9 +1072,29 @@ - (AWSS3TransferUtilityUploadSubTask * ) hydrateMultiPartUploadSubTask:(NSMutabl
         }
         AWSDDLogDebug(@"Request headers:\n%@", request.allHTTPHeaderFields);
         
-        NSURLSessionUploadTask *uploadTask = [self.session uploadTaskWithRequest:request
-                                                                        fromFile:[NSURL fileURLWithPath:transferUtilityUploadTask.file]];
-        transferUtilityUploadTask.sessionTask = uploadTask;
+        NSURLSessionUploadTask *nsURLUploadTask = nil;
+        NSString *exceptionReason = @"";
+        @try {
+            nsURLUploadTask  = [self.session uploadTaskWithRequest:request
+            fromFile:[NSURL fileURLWithPath:transferUtilityUploadTask.file]];
+        }
+        @catch (NSException *exception) {
+            AWSDDLogDebug(@"Exception in upload task %@", exception.debugDescription);
```

Comment:
Log this as error instead of debug?

Reply:
Changed to `AWSDDLogWarn` because we are not doing anything inside the exception block other than passing the value to the error.